monitoring-dashboard/distribution/src/main/resources/bin/yajsw/wrapper.conf

```diff
@@ -0,0 +1,96 @@
+# Tested with open jdk 11.05 & yajsw-stable-12.14
+#********************************************************************
+# working directory
+#********************************************************************
+wrapper.working.dir=${wso2mi_dashboard_home}\\
+# Java Main class.
+# YAJSW: default is "org.rzo.yajsw.app.WrapperJVMMain"
+# DO NOT SET THIS PROPERTY UNLESS YOU HAVE YOUR OWN IMPLEMENTATION
+# wrapper.java.mainclass=
+#********************************************************************
+# tmp folder
+# yajsw creates temporary files named in_.. out_.. err_.. jna..
+# per default these are placed in jna.tmpdir.
+# jna.tmpdir is set in setenv batch file to <yajsw>/tmp
+#********************************************************************
+wrapper.tmp.path = ${jna_tmpdir}
+#********************************************************************
+# Application main class or native executable
+# One of the following properties MUST be defined
+#********************************************************************
+# Java Application main class
+wrapper.java.app.mainclass=org.wso2.ei.dashboard.bootstrap.Bootstrap
+# Log Level for console output.  (See docs for log levels)
+wrapper.console.loglevel=INFO
+# Log file to use for wrapper output logging.
+wrapper.logfile=${wrapper_home}\/log\/wrapper.log
+# Format of output for the log file.  (See docs for formats)
+#wrapper.logfile.format=LPTM
+# Log Level for log file output.  (See docs for log levels)
+#wrapper.logfile.loglevel=INFO
+# Maximum size that the log file will be allowed to grow to before
+#  the log is rolled. Size is specified in bytes.  The default value
+#  of 0, disables log rolling by size.  May abbreviate with the 'k' (kB) or
+#  'm' (mB) suffix.  For example: 10m = 10 megabytes.
+# If wrapper.logfile does not contain the string ROLLNUM it will be automatically added as suffix of the file name
+wrapper.logfile.maxsize=10m
+# Maximum number of rolled log files which will be allowed before old
+#  files are deleted.  The default value of 0 implies no limit.
+wrapper.logfile.maxfiles=10
+# Title to use when running as a console
+wrapper.console.title="WSO2 Micro Integrator Dashboard"
+#********************************************************************
+# Wrapper Windows Service and Posix Daemon Properties
+#********************************************************************
+# Name of the service
+wrapper.ntservice.name=WSO2MI_DASHBOARD
+# Display name of the service
+wrapper.ntservice.displayname=WSO2 Micro Integrator Dashboard 4.0.1
```

Comment:
since this is filtered true, won't parameters work for 4.0.1

Reply:
Added parameters